test/verify/check-packagekit

```diff
@@ -208,21 +210,40 @@ class TestUpdates(MachineCase):
         b.wait_in_text("#state", "Applying updates")
         b.wait_present("#app div.progress-bar")
 
-        # should have succeeded and show restart page; cancel
+        # should have succeeded and show restart page
         b.wait_present("#app .container-fluid h1")
         b.wait_in_text("#app .container-fluid h1", "Restart Recommended")
+
+        # history on restart page should the two security updates
+        b.wait_present(".expander-title span")
+        b.click(".expander-title span")
+        b.wait_present("ul")
+        b.wait_present("ul .tooltip-ct-outer:nth-child(2) li")
+        self.assertEqual(b.text("ul .tooltip-ct-outer:nth-child(1) li"), "secdeclare")
```

Comment:
I wonder if it makes sense to write a convenience function for this, in case the format changes.
```python
def hasUpdates(parent, updates):
    b.wait_present(parent)
    b.wait_present("{0} .tooltip-ct-outer:nth-child({1}) li".format(parent, len(updates))
    ...
```
(+ loop over the updates and a final `assertFalse`)

then you could just use `hasUpdates("ul", ["secdeclare", "secparse"])` et cetera